javascript-timers/interval.js

```diff
@@ -0,0 +1,15 @@
+var $h1 = document.querySelector('h1');
+
+function countDownEarth() {
+  if ($h1.textContent === '4') {
+    $h1.textContent = '3';
+  } else if ($h1.textContent === '3') {
+    $h1.textContent = '2';
+  } else if ($h1.textContent === '2') {
+    $h1.textContent = '1';
+  } else if ($h1.textContent === '1') {
+    $h1.textContent = '~Earth Beeeelooowww Us~';
+  }
```

Comment:
Within this final else if statement we are going to want to cancel setInterval from running. 

If you added a console log at the top of `countDownEarth` you would see that setInterval is still running after you have displayed your final text. You will need to clear that with `clearInterval`

```
function countDownEarth() { 
  console.log('i am still running');
  if ($h1.textContent === '4') {
    $h1.textContent = '3';
  } else if ($h1.textContent === '3') {
    $h1.textContent = '2';
  } else if ($h1.textContent === '2') {
    $h1.textContent = '1';
  } else if ($h1.textContent === '1') {
    $h1.textContent = '~Earth Beeeelooowww Us~';
    // call clearInterval() and pass in timer id. 
  }
  ```

Reply:
ah i see-- i put a console log at the end of it and i didnt see anything in the console, so i thought it was in the clear haha